maths/fermat_little_theorem.py

```diff
@@ -1,12 +1,24 @@
-# Python program to show the usage of Fermat's little theorem in a division
-# According to Fermat's little theorem, (a / b) mod p always equals
-# a * (b ^ (p - 2)) mod p
```

Comment:
This is the original definition the author was working with. It's very convoluted and a is unnecessary in this equation (appears once on both LHS and RHS)